src/main/java/com/woowacamp/soolsool/core/liquor/dto/response/PageLiquorResponse.java

```diff
@@ -0,0 +1,48 @@
+package com.woowacamp.soolsool.core.liquor.dto.response;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.woowacamp.soolsool.core.liquor.domain.Liquor;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.Getter;
+import org.springframework.data.domain.Pageable;
+
+@Getter
+public class PageLiquorResponse {
+
+    private final boolean hasNext;
+    private final Long nextCursorId;
+    private final List<LiquorElementResponse> liquors;
+
+    @JsonCreator
+    public PageLiquorResponse(
+        final boolean hasNext,
+        final Long nextCursorId,
+        final List<LiquorElementResponse> liquors
+    ) {
+        this.hasNext = hasNext;
+        this.nextCursorId = nextCursorId;
+        this.liquors = liquors;
+    }
+
+    public static PageLiquorResponse of(
+        final Pageable pageable,
+        final List<Liquor> liquors
+    ) {
+        final List<LiquorElementResponse> liquorElements = liquors.stream()
+            .map(LiquorElementResponse::of)
```

Comment:
```suggestion
            .map(LiquorElementResponse::from)
```

of 메서드는 삭제해도 될 것 같아요!

Reply:
마자요 ...